test/dashboard-new.test.js

```diff
@@ -0,0 +1,15 @@
+import { html, fixture, expect } from '@open-wc/testing';
+import { stub } from 'sinon';
+import '../src/dashboard/Dashboard-new.js';
+
+describe('Dashboard New details', () => {
```

Comment:
this component renders 4 <dashboard-menu> as we can see in the code. So, write a test to check if they are indeed in DOM.